.travis.yml

```diff
@@ -18,11 +16,16 @@ jobs:
   include:
     - php: 7.2
       env: COMPOSER_FLAGS="--prefer-lowest"
+    - php: 7.3
```

Comment:
Just a thought: instead of splitting PHP 7.3 and 7.4 and using it for different purposes, why not say that the latest stable PHP version is used with `COMPOSER_FLAGS` and 'LINTER_RUN`? That way we only have to maintain one entry.